CMakeLists.txt

```diff
@@ -1,23 +1,90 @@
-cmake_minimum_required(VERSION 3.9.0)
```

Comment:
Do we still need this file?

Reply:
You are right, I will remove all cmake related stuff and push a commit.

Reply:
Thanks!